<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node pkg="supervisor" type="supervisor.py" name="supervisor">
	<param name="command_controller_topic"             value="/start_stop_controller"/>
	<param name="path_topic"                           value="/race_course"/>
	<param name="leader_path_topic"                    value="/leader_path"/>
	<param name="danger_topic"                         value="/danger"/>
	<param name="emergency_stop_topic"                 value="/emergency_stop"/>
	<param name="waypoint_topic"			   value="/BOSS/pose"/>
	<param name="command_controller_follow_topic"      value="/start_stop_follow"/>
	<param name="selected_controller"		   value="pure_pursuit"/>
	<param name="obstacles_topic"			   value="/obstacles"/>
	</node>
</launch>


